### DISCUSSION TITLE
Haade recherche des contributeurs - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#22] [Comments]
# Haade recherche des contributeurs - Haade.fr

En pleine expansion nous recherchons de nouveaux contributeurs

https://haade.fr/fr/blog/haade-est-a-la-recherche-de-nouveaux-contributeurs

**Comment by Doubledom45** [comment]
Bjr..
Peux tu modifier ton lien  page tests de produits url => https://haade.fr/fr/categorie/tests/
Il ne faut pas le / de fin !

**Reply by haade-administrator** [reply]
👍🏻
